frontend/src/metabase/query_builder/hooks/use-query-aggregations.ts

```diff
@@ -0,0 +1,39 @@
+import * as Lib from "metabase-lib";
+
+import { STAGE_INDEX } from "./constants";
+import type { UpdateQueryHookProps } from "./types";
+
+export const useQueryAggregations = ({
+  query,
+  onQueryChange,
+}: UpdateQueryHookProps) => {
+  const aggregations = Lib.aggregations(query, STAGE_INDEX);
+
+  const aggregationData = aggregations.map((aggregation, aggregationIndex) => {
```

Comment:
This hook can be used later for providing the name of the aggregation and the `remove` helper method for SDK usage, so we can put these values in a list, set of badges, etc in the future